### ISSUE TITLE
Double clicking on my friends dm makes DDPP crash.

### ISSUE THREAD

**Issue by ghost** [ramzialhaddadtm/discord-data-package-viewer-parser#3]
So like in tittle.
Trying to load up any conversation makes me crash.

**Comment by AvianAeternum** [comment]
Confirmed

**Comment by EnderShadowborne** [comment]
Also confirmed, two years later

**Comment by ramzialhaddad** [comment]
This program is extremely old. There are much more modern, maintained and feature rich discord package viewers. A quick Google search will land you on several websites and programs that offer this functionality. Just ensure they are trustworthy since your package contains sensitive information.

**Comment by EnderShadowborne** [comment]
Not really, I've tried a few of them and, I gotta say, they don't exactly do it for me... One of them even refuses to finish parsing, I even left it overnight, too!
